app/registries/foreman/access_permissions.rb

```diff
@@ -8,7 +8,7 @@
     map.permission :my_account, {
       :users => [:edit],
       :notification_recipients => [:index, :update, :destroy, :update_group_as_read, :destroy_group],
-      :"api/v2/table_preferences" => [:show, :create, :edit, :delete, :index],
+      :"api/v2/table_preferences" => [:show, :create, :edit, :update, :delete, :index],
```

Comment:
```suggestion
      :"api/v2/table_preferences" => [:show, :create, :update, :destroy, :index],
```

The controller doesn't have `edit` and `delete` methods, but it does have `destroy` method